src/applications/check-in/components/TravelPayReimbursementLink.jsx

```diff
@@ -1,14 +1,25 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import { useTranslation } from 'react-i18next';
+import recordEvent from 'platform/monitoring/record-event';
+
+import { createAnalyticsSlug } from '../utils/analytics';
 import ExternalLink from './ExternalLink';
 
 function TravelPayReimbursementLink() {
   const { t } = useTranslation();
+  const handleClick = useCallback(() => {
```

Comment:
What would you think of passing the event ID instead and conditionally recording the event in the ExternalLink component if it's given one? Seems like we're likely to want info on other link clicks and feels a little nicer than passing around a handler to me. (likely only a matter of taste though)

Reply:
yeah that makes sense to me!